### ISSUE TITLE
IOP Science Template

### ISSUE THREAD

**Issue by lucas-johnson** [rstudio/rticles#428]
Would love to see a template for IOP Science. Info and links to template: https://publishingsupport.iopscience.iop.org/questions/latex-template/

By filing an issue to this repo, I promise that

- [ X] I have fully read the issue guide at https://yihui.name/issue/.
- [ X] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('rticles')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/rticles')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [ X] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by cderv** [comment]
Hi, 

Thank you for the suggestion. Most of **rticles** templates are community contributed. That is why I have added the labels so that anyone know this is to pick up. 

If you would like to try, we welcome a PR ! We tried to provide the relevant informations in https://github.com/rstudio/rticles#how-to-contribute about how to do it, and you get all other formats as examples (see table in README). 

Otherwise, anyone is welcome to give it a try ! A nice project to better understand **rmarkdown** and Pandoc.
